kcidb/io/schema/__init__.py

```diff
@@ -1,8 +1,11 @@
 """Kernel CI reporting I/O schema"""
 
 from kcidb.io.schema import v1
+from kcidb.io.schema import v2
```

Comment:
Do we need to keep a copy of each schema version? Are we actually planning to have backwards-compatibility support for this PoC?

Reply:
Yes. Apart from letting all submitters upgrade gradually, we will use this when upgrading the datasets to new schema (download, convert, upload).

Reply:
I see.  Is there an automated way of migrating a dataset from v1 to v2 from a db dump?

Reply:
Not completely automated, but there are instructions, in README.md.